_posts/2021-03-29-changelog-144.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #144"
+date: 2021-03-29 17:00:00 +0300
+---
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6956 -->
+* Increase minimal supported rustc version to 1.41.0
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6907 -->
+* Provide debugging with LLDB in [Rider for Unreal Engine](https://www.jetbrains.com/lp/rider-unreal/) 2021.1
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6799 -->
+* Complete dependencies by subwords in `Cargo.toml` when using experimental local crates index. This feature is disabled by default
+  for now. To use it, enable `org.rust.crates.local.index` experimental feature
+
+  {% include gif-img.html path="/assets/posts/changelog-144/6799" w="600px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6893 -->
+* Import unresolved types from a function signature after performing `Extract Function` refactoring (`Refactor | Extract Method` or
+  <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>M</kbd>)
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6947 -->
+* Support import aliases in `Move` refactoring (`Refactor | Move` or <kbd>F6</kbd>)
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6966 -->
+* Automatically select the default project Rust toolchain from `Preferences | Languages & Frameworks | Rust` when creating a new project
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7001 -->
+* (maybe group together with other const generic things?) Support [const generics](https://rust-lang.github.io/rfcs/2000-const-generics.html) in `Wrong generic arguments number` ([E0107](https://doc.rust-lang.org/error-index.html#E0107)) inspection
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6066 -->
+*  Use [`todo!()`](https://doc.rust-lang.org/std/macro.todo.html) instead of [unimplemented!()](https://doc.rust-lang.org/std/macro.unimplemented.html) in function bodies generated by `Implement members` action (by [@KarelPeeters])
```

Comment:
```suggestion
*  Use [`todo!()`](https://doc.rust-lang.org/std/macro.todo.html) instead of [`unimplemented!()`](https://doc.rust-lang.org/std/macro.unimplemented.html) in function bodies generated by `Implement members` action (by [@KarelPeeters])
```

Reply:
Not only by `Implement members`